xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/ExportAction.java

```diff
@@ -70,6 +75,93 @@
  */
 public class ExportAction extends XWikiAction
 {
+    /**
+     * Used to separate page arguments in excludes parameter
+     */
+    private static String PAGE_SEPARATOR = "&";
+
+    /**
+     * Manage the arguments of the export and provides them in an object to
+     * simplify their usage in the different methods.
+     *
+     * @since 10.9RC1
+     */
+    private class ExportArguments {
+        /**
+         * Represent the pages to be included and excluded:
+         * keys are pattern of pages to include, values are list of pages to exclude
+         */
+        private Map<String, List<String>> exportPages;
+
+        /**
+         * Name of the export
+         */
+        private String name;
+
+        /**
+         * Description of the export
+         */
+        private String description;
+
+        ExportArguments(XWikiContext context) throws XWikiException
+        {
+            XWikiRequest request = context.getRequest();
+
+            this.description = request.get("description");
+
+            this.name = request.get("name");
+            if (StringUtils.isBlank(name)) {
+                this.name = context.getDoc().getFullName();
+            }
+
+            String[] pages = request.getParameterValues("pages");
+            String[] excludes = request.getParameterValues("excludes");
+
+            this.exportPages = new HashMap<>();
```

Comment:
Just a detail: I would use a LinkedHashMap here to keep the same order than what you got in input.

Reply:
Fixed.